src/main/scala/is/hail/expr/Type.scala

```diff
@@ -2202,14 +2202,27 @@ final case class TStruct(fields: IndexedSeq[Field], override val required: Boole
     (t, selectF)
   }
 
+  var missingIdx: Array[Int] = _
+  private val _nMissing: Int = {
```

Comment:
`private` is enough.  Don't use leading or trailing underscores in names in Scala.  I agree with this post: https://stackoverflow.com/a/9656359/431282